### DISCUSSION TITLE
Support of OV5640

### DISCUSSION THREAD

**Discussion by Brueller1982** [jomjol/AI-on-the-edge-device#3062] [Ideas]
Hallo,

zurzeit wird meines Erachtens nach das Kamera Modul OV5640 nicht unterstützt. 
Stimmt das? Gibt es dafür Gründe? 
Wäre es möglich auch dieses Kameramodul zu unterstützen, nicht nur OV2460 und OV3660.

Die Kamera Komponente ESP32-Camera supported schon das OV5640.

**Comment by SybexX** [comment]
https://github.com/jomjol/AI-on-the-edge-device/issues/2162#issuecomment-2094309958
